samples/layouts/stepper/stepper-overview-example/index.html

```diff
@@ -0,0 +1,190 @@
+          <igc-input required label="Phone Number" type="text" name="phoneNumber"></igc-input>
+          <igc-input required label="Email Address" type="email" name="email"></igc-input>
+          <igc-checkbox required>
+            I confirm that I am authorized to borrow on behalf of this business.<span class="required">*</span>
+          </igc-checkbox>
+          <igc-checkbox required>
+            I agree with the<span class="required">*</span><a class="terms-and-conditions">Terms and Conditions</a>
+          </igc-checkbox>
+        </igc-form>
+        <div class="card-wrapper selected-card-holder"></div>
+      </div>
+      <br />
+      <div>
+        <igc-button onclick="stepper.prev()">PREVIOUS</igc-button>
+        <igc-button class="next" onclick="stepper.next()" disabled>NEXT</igc-button>
+      </div>
+    </igc-step>
+
+    <igc-step>
+      <span slot="title">Shipping Details</span>
+      <div class="sample-row">
+        <igc-form class="flex-form">
+          <igc-input required label="First Name" type="text" name="firstName"></igc-input>
+          <igc-input required label="Last Name" type="text" name="lastName"></igc-input>
+          <igc-input required label="Mailing Address" type="text" name="mailingAddress"></igc-input>
+          <div class="form-row">
+            <igc-input required label="City" type="text" name="city"></igc-input>
+            <igc-select label="State" name="state" required>
+              <igc-select-item value="nevada">Nevada</igc-select-item>
+              <igc-select-item value="newYork">New York</igc-select-item>
+              <igc-select-item value="newJersey">New Jersey</igc-select-item>
+              <igc-select-item value="ohio">Ohio</igc-select-item>
+              <igc-select-item value="texas">Texas</igc-select-item>
+            </igc-select>
+            <igc-mask-input required label="ZIP Code" type="text" name="zip">
+              </igc-input>
+          </div>
+        </igc-form>
+        <div class="card-wrapper selected-card-holder"></div>
+      </div>
+      <br />
+      <div>
+        <igc-button onclick="stepper.prev()">PREVIOUS</igc-button>
+        <igc-button class="next" onclick="stepper.next()" disabled>NEXT</igc-button>
+      </div>
+    </igc-step>
+
+    <igc-step>
+      <span slot="title">Confirmation</span>
+      <div class="success-message">
+        <div class="success-marker">
+          <igc-icon size="large" name="check" collection="material"></igc-icon>
+        </div>
+        <div class="success-text">
+          <h2 class="sample-step-title">Your application for a business credit card has been sent
+            successfully!</h2>
+          <p>We will contact you within a few days.</p>
+        </div>
+      </div>
+      <div class="card-wrapper selected-card-holder card-horizontal"></div>
+      <br />
+      <igc-button onclick="stepper.reset()">RESET</igc-button>
```

Comment:
This should not only reset the stepper but clear all of the inputs and the selected card as well.